daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala

```diff
@@ -510,7 +510,7 @@ object Repl {
           case None =>
             println("Error: " + id + " not found.")
             None
-          case Some(DValue(_, _, body, _)) =>
+          case Some(DValue(_, _, body, true)) =>
```

Comment:
Hm, is this `true` supposed to be here? This `buildExpr` function doesn't strike me as being for test scenarios only?

Reply:
It’s only used for testing and profiling scenarios.

Reply:
I renamed the function

Reply:
Thanks! 👍